viroconcom/plot.py

```diff
@@ -0,0 +1,520 @@
+    y_median : ndarray of floats
+        The 50-percentile contour's coordinates in the y-direction.
+    x_bottom : ndarray of floats
+        The bottom percentile contour's coordinates in the x-direction.
+    y_bottom : ndarray of floats
+        The bottom percentile contour's coordinates in the y-direction.
+    x_upper : ndarray of floats
+        The upper percentile contour's coordinates in the x-direction.
+    y_upper : ndarray of floats
+        The upper percentile contour's coordinates in the y-direction.
+    ax : Axes
+        Axes of the figure where the contour should be plotted.
+    x_label : str, optional (defaults to None)
+        Label for the x-axis.
+    y_label : str, optional (defaults to None)
+        Label for the y-axis.
+    contour_labels : list of str, optional (defaults to [None, None, None])
+        Label for the environmental contours that will be used in the legend.
+    plotted_sample : PlottedSample, optional
+        If provided, this sample is plotted together with the contours.
+    """
+    plot_contour(x=x_median,
+                 y=y_median,
+                 ax=ax,
+                 x_label=x_label,
+                 y_label=y_label,
+                 line_style='b-',
+                 contour_label=contour_labels[0],
+                 plotted_sample=plotted_sample)
+    plot_contour(x=x_bottom,
+                 y=y_bottom,
+                 contour_label=contour_labels[1],
+                 line_style='r--',
+                 ax=ax)
+    plot_contour(x=x_upper,
+                 y=y_upper,
+                 contour_label=contour_labels[2],
+                 line_style='r--',
+                 ax=ax)
+
+
+def plot_wave_breaking_limit(ax, bottom_tz=0, upper_tz=20, steps=100):
+    """
+    Plots the wave breaking limit on a given axes.
+
+    Assumes that x = zero-up-crossing period and y = sig. wave height.
+
+    Parameters
+    ----------
+    ax : Axes
+    bottom_tz : float (defaults to 0)
+        Bottom limit for the curve.
+    upper_tz : float (defaults to 20)
+        Upper limit for the curve.
+    steps : int (defaults to 100)
+        Number of points that are plotted on the curve.
+    """
+    tz_lim = np.linspace(bottom_tz, upper_tz, steps)
+    hs_lim = hs_from_limiting_sig_wave_steepness(tz_lim)
+    ax.plot(tz_lim, hs_lim, linestyle='-.', color=[0.5, 0.5, 0.5])
+
+
+def hs_from_limiting_sig_wave_steepness(tz):
+    """
+    Calculates highest Hs value for a given Tz based on wave steepness.
+
+    The calculaion uses the 'limiting significant wave steepness' described in
+    DNV GL's DNV-GL-RP-C205:2017 (section 3.5.3.5. and 3.5.4)
+
+    Parameters
+    ----------
+    tz : ndarray of floats
+        Zero-up-crossing period in seconds.
+    Returns
+    -------
+    hs : ndarray of floats
+        Significant wave height in meters.
+    """
+    TZLOW_STEEPNESS_VALUE = 0.1 # is 1/10 in DNVG-GL-RP-C205:2017, 3.5.4

```

Comment:
Uppercase usually indicates a global variable.

Reply:
Is there a way to clarify that this is a local sontant using notation?